lib/models/message.ts

```diff
@@ -1,4 +1,5 @@
 import * as t from "io-ts";
+import { PathReporter } from "io-ts/lib/PathReporter";
```

Comment:
you can use [this reporter](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/utils/validation_reporters.ts#L42) that has a more readable output, it's the one we use for the API errors